templates/OperationOptionsSnippet.stencil

```diff
@@ -2,12 +2,10 @@
 if let options = options {
     // Query options
     {% for param in op.params.query.optional %}
-    if let {{ param.key }} = options.{{ param.key }} {
-        {% include "OperationOptionsDecodingSnippet.stencil" %}
-        queryParams.append("{{ param.key }}", {{ param.value}})
-    }
+    {% if param.source == "options" %}
+    {% include "OperationOptionalQueryParamSnippet.stencil" %}
+    {% endif %}
```

Comment:
This feels wrong. We should not have to check that the source is options.

Reply:
Removed